src/proto/primitive.proto

```diff
@@ -3,13 +3,56 @@ syntax = "proto3";
 package TbotsProto;
 
 import "proto/geometry.proto";
+import "google/protobuf/descriptor.proto";
```

Comment:
more of a general question than a comment, but does the "google" part mean we're importing this from somewhere on google or do we have a folder somewhere called google (I tried looking for one but couldn't find anything)?

Reply:
yeah it's not obvious...
at the top of the proto/BUILD folder we import `load("@rules_proto//proto:defs.bzl", "proto_library")`, which is built into bazel and it automatically gives us `google/protobuf/descriptor.proto`, which can be found in the bazel-out folder, once we built